server/sonar-db-dao/src/test/java/org/sonar/db/user/UserDbTester.java

```diff
@@ -125,6 +126,18 @@ public GroupDto insertGroup(GroupDto dto) {
     return dto;
   }
 
+  public GroupDto insertDefaultGroup(GroupDto dto) {
```

Comment:
For me, it should be done explicitly if you want to test default group, otherwise DbTester will contain too many hidden logic => For instance how to test the creation of the default group ?